src/main/java/seedu/sudohr/logic/commands/eventCommands/DeleteEmployeeFromEventCommand.java

```diff
@@ -0,0 +1,89 @@
+package seedu.sudohr.logic.commands.eventcommands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+import java.util.Set;
+
+import seedu.sudohr.commons.core.index.Index;
+import seedu.sudohr.logic.commands.Command;
+import seedu.sudohr.logic.commands.CommandResult;
+import seedu.sudohr.logic.commands.exceptions.CommandException;
+import seedu.sudohr.model.Model;
+import seedu.sudohr.model.events.Event;
+import seedu.sudohr.model.events.EventContainsEmployeePredicate;
+import seedu.sudohr.model.person.Person;
+
+/**
+ * Deletes a person from a specific event in sudohr book.
+ */
+public class DeleteEmployeeFromEventCommand extends Command {
+    public static final String COMMAND_WORD = "deleteEmployeeEvent";
+
+    public static final String MESSAGE_PERSON_DOES_NOT_EXIST = "This user does not exists in the event in sudohr book";
```

Comment:
Should be "This user does not exists in the event in SudoHR"

Reply:
Found this in the rest of the commands, I think you're probably planning to fix this but just commenting this here in case

Reply:
@4ndrelim , I think this one can leave it separate from your current issue for now. Although it is important to settle this early so we don't have a repeat incident, I'm thinking we can split this consistency problem of "sudohr book" based on who is handling which component, so we don't have the crazy merge conflict problem again.